data/core/macros/optional_unit_advancements.cfg

```diff
@@ -57,11 +57,6 @@
     {ENABLE_ADVANCEMENT "Great Wolf" "Direwolf" (set_experience=65)}
 #enddef
 
-#define ENABLE_NIGHTBLADE
```

Comment:
I am not sure deleting a core macro just out of the blue is a good approach. There are UMC authors using it still. So their campaigns will break instantly. Add a dummy macro here so their works are not hampered.

Reply:
Or, rather move it to the deprecated section.

Reply:
Alright, moved it to deprecated_utils.

Put it at the end of the file; let me know if you think there is a better spot for it within deprecated-utils.

Reply:
That's okay. Are you sure it wont throw any error? like the deprecated-utils.cfg is read before the advancements one so the two generalised advancement/disadvancement macros aren't read before.

Reply:
I did not see any errors in the log files. But now that I think about it, there are no calls to ENABLE_NIGHTBLADE anywhere in current mainline so any could-be errors may not have been thrown. I applied the fix suggested by CelticMinstrel below.